ci-operator/jobs/openshift-qe/ocpqe-security-tools/openshift-qe-ocpqe-security-tools-main-presubmits.yaml

```diff
@@ -0,0 +1,126 @@
+presubmits:
```

Comment:
WE could remove this one from commit unless you want to keep this around for testing purposes and harmless.

Reply:
I would like to keep this around for testing images quickly in the future.